config/environments/development.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false #KIRSTIN for Docker
```

Comment:
I just took some time testing with that parameter. I used this branch, build the image as it is with 0b8274e and started a container. If I try to reach a page that is not available, I just get that general error message that the page I was looking for doesn't exist. This is the case on the local machine as well as on a remote machine accessing the web server through the network. 

On the other hand, changing that parameter to true and rebuilding the image changes that behaviour to display more useful information. Is that the same you would expect or did you experienced something different?

`config.consider_all_requests_local = false`:

<img width="1440" alt="consider_all_requests_local false" src="https://cloud.githubusercontent.com/assets/7300329/14107243/a5f4d3f2-f602-11e5-8398-fab645c153db.png">

<hr/>

`config.consider_all_requests_local = true`:

![consider_all_requests_local true](https://cloud.githubusercontent.com/assets/7300329/14107244/a6a85eea-f602-11e5-8488-b26d9488643a.png)